app/models/errors.rb

```diff
@@ -1,3 +1,5 @@
 module Errors
   class ContractError < StandardError; end
+
+  class ClientDetailsMismatchError < StandardError; end
```

Comment:
Does `client` here refer to the person whose details are being used for searching (as opposed to the calling service)? If so, there might be a better name for this - maybe `ApplicantDetailsMismatchError`,  `UserDetailsMismatchError`, or `IndividualDetailsMismatchError` (although none of those are perfect either...)

Reply:
How about `CitizenDetailsMismatchError`? 
I have been thinking we should probably start a `docs` folder with some guidance for new users and, maybe, we should include a definitions list that we can populate/adhere to! 
What do you think?

Reply:
I like `CitizenDetailsMismatchError`. Makes it obvious who the details belong to but is also still nicely generic.

I think a guidance and a glossary would be a good idea, especially as the service is going to be used across different teams in the department and we're all likely to have different vocabularies.

Reply:
I have rebased on main and re-named the error. I'll add a ticket for the documentation as we are nearing the point of MVP :+1: